app/controllers/case_assignments_controller.rb

```diff
@@ -1,4 +1,6 @@
 class CaseAssignmentsController < ApplicationController
+  # NOTE: I don't know what auth levels to use here, but am pretty certain it needs SOMETHING.
+  before_action :authenticate_user!
```

Comment:
This is taken care of by `must_be_admin` on the line below

Reply:
Are you certain? It was failing the spec beforehand, but passed afterwards.

Reply:
Hmm well it *should* be taken care of by `must_be_admin` but it sounds like we have a bug! Thanks for the catch :)